viam-cartographer/src/carto_facade/map_builder.h

```diff
@@ -72,8 +72,10 @@ class MapBuilder {
 
     // AddSensorData adds sensor data to cartographer's internal state
     // throws if adding sensor data fails.
-    // Currently assumes the sensor data is coming from the lidar
-    void AddSensorData(cartographer::sensor::TimedPointCloudData measurement);
+    void AddSensorData(const std::string &sensor_id,
```

Comment:
[note] Different typing of these functions allow them to have the same name

Reply:
Yep, this is what sensor_id is needed for